docs/intro-react-native-components.md

```diff
@@ -0,0 +1,100 @@
+---
+id: intro-react-native-components
+title: Core Components and Native Components
+---
+
+React Native is an open source framework for building Android and iOS applications using [React](https://reactjs.org/) and the app platform’s native capabilities. With React Native, you use JavaScript to access your platform’s APIs as well as to describe the appearance and behaviour of your UI using React components, bundles of reusable, nestable code. You can learn more about React in the next section. But first, let's cover how components work in React Native.
+
+## Views and mobile development
+
+In Android and iOS development, a **view** is the basic building block of UI: a small rectangular element on the screen which can be used to display text, images, or respond to user input. Even the smallest visual elements of an app, like a line of text or a button, are kinds of views. Some kinds of views can nest other views. It’s views all the way down!
+
+## Native Components
+
+In Android development, you write views in Java or Kotlin; in iOS development, you use Objective-C or Swift. With React Native, you can invoke these views with JavaScript using React components. At runtime, React Native creates the corresponding Android and iOS views for those components. Because React Native components are backed by the same views as Android and iOS, React Native apps look, feel, and perform like any other apps. We call these platform-backed components Native Components.
+
+## Core Components
+
+React Native comes with a suite of built-in components ready for you to use. These Core Components include:
+
+| React Native UI Component | Android View   | iOS View         | Web Analog               | Description                                                                                           |
```

Comment:
Our table rendering looks kinda off. We should figure out how to give this table the full width available, or a min-width.